codeguru_profiler_agent/sdk_reporter/profile_encoder.py

```diff
@@ -42,6 +46,15 @@ def _get_module_path(file_path, sys_paths):
     return module_path
 
 
+def _remove_prefix_path(module_path, sys_paths):
+    current_path = str(Path().absolute())
+    if current_path in module_path:
+        return module_path.replace(current_path, "").replace("/.", "")
```

Comment:
Should we not replace "/./" by "/" ?

Reply:
also that would not work in windows because the "/" would have been replaced already right? Are we sure we cannot have that problem in windows?

Reply:
Sounds better to replace /./, I'll update. I didn't test anything on windows. The existing unit tests with Windows passed, but I didn't add anything new.